.github/workflows/packaging.yml

```diff
@@ -0,0 +1,89 @@
+name: packaging
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  package:
+    # Skip pull request jobs when the source branch is in the same
+    # repository.
+    if: |
+      github.event_name == 'push' ||
+      github.event.pull_request.head.repo.full_name != github.repository
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        platform:
+          - { os: 'debian', dist: 'stretch' }
+          - { os: 'debian', dist: 'buster' }
+          - { os: 'debian', dist: 'bullseye' }
+          - { os: 'el', dist: '7' }
+          - { os: 'el', dist: '8' }
+          - { os: 'fedora', dist: '30' }
+          - { os: 'fedora', dist: '31' }
+          - { os: 'fedora', dist: '32' }
+          - { os: 'fedora', dist: '33' }
+          - { os: 'fedora', dist: '34' }
+          - { os: 'fedora', dist: '35' }
+          - { os: 'fedora', dist: '36' }
+          - { os: 'ubuntu', dist: 'xenial' }
+          - { os: 'ubuntu', dist: 'bionic' }
+          - { os: 'ubuntu', dist: 'focal' }
+          - { os: 'ubuntu', dist: 'groovy' }
+          - { os: 'ubuntu', dist: 'hirsute' }
+          - { os: 'ubuntu', dist: 'impish' }
+          - { os: 'ubuntu', dist: 'jammy' }
+
+    env:
+      OS: ${{ matrix.platform.os }}
+      DIST: ${{ matrix.platform.dist }}
+
+    steps:
+      - name: Clone the module
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Clone the packpack tool
+        uses: actions/checkout@v2
+        with:
+          repository: packpack/packpack
+          path: packpack
+
+      - name: Fetch tags
+        # Found that Github checkout Actions pulls all the tags, but
+        # right it deannotates the testing tag, check:
+        #   https://github.com/actions/checkout/issues/290
+        # But we use 'git describe ..' calls w/o '--tags' flag and it
+        # prevents us from getting the needed tag for packages version
+        # setup. To avoid of it, let's fetch it manually, to be sure
+        # that all tags will exist always.
+        run: git fetch --tags -f
+
+      - name: Create packages
+        run: ./packpack/packpack
+
+      - name: Deploy packages
+        # We need this step to run only on push with tag.
+        if: ${{ github.event_name == 'push' && startsWith(github.ref, 'refs/tags/') }}
```

Comment:
I reckon we should. I've extended the list of triggers to:

```yml
on:
  pull_request:
  workflow_dispatch:
  push:
    tags:
      - '*'
```
After that we'll have an opportunity to test package building on each commit to any branch if the PR was created. `workflow_dispatch` will allow to trigger building manually. Anyway, publishing of the package will start only for pushing tags.

Reply:
Ah, I see, it is on any pull request: from the base repository branch or from a fork. But how about `master`? I think we should run the testing on pushing to `master` as well.

Reply:
I think it's better to tag more often, because we don't use this practice widely, for example, we don't publish `Tarantool` on each commit to `master`. If you insist, we can push packages on each commit to `master`. In the suggested configuration we have only testing builds on each push.

Reply:
I agree, enabled testing on pushing to `master`